### ISSUE TITLE
Release on mavenCentral

### ISSUE THREAD

**Issue by perryrh0dan** [BoD/android-wear-color-picker#8]
Hi,

First thank you so much for this great library.
Are there any plans to release this library to the mavenCentral repository. Due to the jcenter shutdown ?

Best regards
Thomas

**Comment by BoD** [comment]
Hi!

Yes, good idea, I will try to find a bit of time soon to do this.

**Comment by perryrh0dan** [comment]
Thx for the quick response. That would be rly great :)

**Comment by yingding** [comment]
Thx, i am also looking forward to this.

**Comment by BoD** [comment]
Hi!  I have just published `v2.2.4` on Maven Central. It is the same as `v2.2.3` but with dependencies updated to the latest versions.

Note: unfortunately, I made a mistake and first published this updated version as `v2.2.3` on Maven Central, and there is no way to un publish it. 

The result is that `v2.2.3` on JCenter and `v.2.2.3` on Maven Central are not exactly the same (the one on Maven Central has the updated dependencies).

I hope this won't cause too much confusion and I am sorry for the mistake.
